dev_guide/expose_service/expose_internal_ip_service.adoc

```diff
@@ -305,7 +307,7 @@ These steps assume that all of the systems are on the same subnet.
 . Restart the network to make sure the network is up.
 +
 ----
-$ service network restart
+# service network restart
```

Comment:
I don't see this on the preview, but it looks like the rest of the procedure just uses $. I normally don't think it's good to bounce around without calling it out. Should the rest just be updated to #? wdyt?

Reply:
I only switched this one since it would need sudo/root privileges, whereas the other commands wouldn't AFAICT.

Reply:
Can call it out though.

Reply:
Related, we'd always talked about putting in some sort of docs conventions topic where this would have been more clear to the reader, but we still have not. :( E.g., this one per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks.